osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -621,7 +624,8 @@ protected override void Update()
                         var panel = setPool.Get(p => p.Item = item);
 
                         panel.Depth = item.CarouselYPosition;
-                        panel.Y = item.CarouselYPosition;
+                        // place the panel at current Y by accounting for the time the panel has spent not updated yet
+                        panel.Y = DrawableCarouselBeatmapSet.YLerp(item.CarouselYPosition, item.ExpectedYPosition, elapsed);
```

Comment:
why is nothing happening with `CarouselYPosition` here? isn't it getting out-of-sync?

Reply:
Renaming it to ``TargetYPosition`` might be the best.

We have already calculated our target Y in ``updateYPositions`` and this won't change until another filtering has happened. This is also just the initial placement for the panel, the drawable itself will keep moving towards the target Y until it is reached.

Reply:
so now it's `Target` and `Expected`? very confusing...

Reply:
``Target`` and ``Current`` make most since, but ``Current`` is only update on filter so it's not really current.

Reply:
then maybe go descriptive? `YPositionOnLastFilter`? mind you that doesn't help this diff *that* much in terms of readability because the logic is half-split between that field and the actual drawable's `Y` anyway...

the overall point is to be able to tell from a glance on the names which value is the stale one and which one are we going towards.